standalone/pherry/src/main.rs

```diff
@@ -172,10 +172,11 @@ struct Args {
     )]
     tip: u64,
     #[structopt(
+        default_value = "5",
```

Comment:
Why `5` rather than `4`?  It should be a power of two.

Reply:
Got it. We need to add more check there.